From cde2c8aca0d925898b322c299b1e480a67e47739 Mon Sep 17 00:00:00 2001
From: Quynh Nguyen <quynh.nguyen.xb@renesas.com>
Date: Tue, 14 Jul 2020 12:59:57 +0700
Subject: [PATCH 108/441] ARM: dts: iwg22m: Remove msiof2_sync pin

msiof2_sync is shared with CAN.
Remove it to prevent conflict with CAN.
With current testing (write/read), msiof2 still work OK without
msiof2_sync (so msiof2_ss2 will be use as chip select signal).

Futher testing/investigating may be needed.

Signed-off-by: khanhluu <khanh.luu.xw@rvc.renesas.com>
Signed-off-by: Quynh Nguyen <quynh.nguyen.xb@renesas.com>
---
 arch/arm/boot/dts/r8a7745-iwg22m.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a7745-iwg22m.dtsi b/arch/arm/boot/dts/r8a7745-iwg22m.dtsi
index ad841ad..c9fb476 100644
--- a/arch/arm/boot/dts/r8a7745-iwg22m.dtsi
+++ b/arch/arm/boot/dts/r8a7745-iwg22m.dtsi
@@ -67,7 +67,7 @@
 
 	msiof2_pins: spi3 {
 		groups = "msiof2_clk", "msiof2_tx", "msiof2_rx",
-			"msiof2_ss2", "msiof2_sync";
+			"msiof2_ss2";
 		function = "msiof2";
 	};
 
-- 
2.7.4

